### PR TITLE
[CAM-11243] Update CAM-ALM API documentation to include abort instance operation

### PR DESCRIPTION
**Pull request by EfrenRey**
@gsanchezu @cristb

### CODE REVIEW

File: Application Lifecycle Management/instances-api.md [hunk 1]
```diff
@@ -2956,3 +2957,44 @@ ElasticBox-Release: 4.0
   "method":"cancel_import"
 }
 ```
+
+## PUT /services/instances/{instance_id}/abort_operation
+
+Abort a processing operation executing scripts on an instance when you give its ID.
```
Comment (gsanchezu) [inline Application Lifecycle Management/instances-api.md]: I would add the preconditions: write access to the instance? it is in processing state and executing a script?

File: Application Lifecycle Management/instances-api.md [hunk 2]
```diff
@@ -2956,3 +2957,44 @@ ElasticBox-Release: 4.0
...
+Content-Type: application/json
+Authorization: Bearer your_json_web_token
+ElasticBox-Release: 4.0
+```
+
+#### URI Parameters
+| Parameter | Type | Description | Req.|
+|-----------|------|------------|----|
+| instance_id | string | Instance id | Yes |
+### Response
```
Comment (gsanchezu) [inline Application Lifecycle Management/instances-api.md]: Please, leave 1 blank line above and below titles

Comment (EfrenRey) [reply Application Lifecycle Management/instances-api.md]: Done

File: Application Lifecycle Management/instances-api.md (hunk 1 above)
Comment (EfrenRey) [reply Application Lifecycle Management/instances-api.md]: Done

**Comment by EfrenRey** [comment]
@jim-phillips Please, could you merge this? Thanks

**Pull request merged**